apps/js_urls/templatetags/js_urls.py

```diff
@@ -0,0 +1,22 @@
+# Django Imports
+from django import template
+from django.utils.safestring import mark_safe
+
+# HTK Imports
+from htk.apps.js_urls.serializer import get_urls_as_json
+from htk.utils import htk_setting
+
+
+register = template.Library()
+
+
+@register.simple_tag
+@mark_safe
+def print_js_urls():
```

Comment:
- Would prefer to put this directly in `htk/tempatetags/**` so that we can just `{% load htk_tags %}`
- I think `print_js_routes`sounds better?

Reply:
Done.